logic/poi_locations.yaml

```diff
@@ -60,7 +60,7 @@
       top: 0
   logic: tavern
 - name: Chicken House
-  hoverText: Chicken House
+  hoverText: Chicken House <img src='/static/image/bomb.png' class='mini'>
```

Comment:
is there no way to make these more dynamic so the html gets injected and you're not typing out the same thing for the bomb image over and over here in the yaml? seems not super ideal, especially if for some reason there was a directory structure or naming convention change down the line